kview/src/main/java/dev/ikm/komet/kview/klfields/KlFieldHelper.java

```diff
@@ -158,48 +149,19 @@ public static List<ObservableField<?>> displayReadOnlySemanticFields(ViewPropert
                 } else if (dataTypeNid == TinkarTerm.BOOLEAN_FIELD.nid()) {
                     KlBooleanFieldFactory klBooleanFieldFactory = new KlBooleanFieldFactory();
                     readOnlyNode = klBooleanFieldFactory.create(observableField, viewProperties.nodeView(), false).klWidget();
+//                } else if (dataTypeNid == TinkarTerm.IMAGE_FIELD.nid()) {
+//                    KlImageFieldFactory imageFieldFactory = new KlImageFieldFactory();
+//                    readOnlyNode = imageFieldFactory.create(observableField, viewProperties.nodeView(), false).klWidget();
+                } else if (dataTypeNid == TinkarTerm.BYTE_ARRAY_FIELD.nid()) {
+                    KlImageFieldFactory imageFieldFactory = new KlImageFieldFactory();
```

Comment:
We would not support currently, but we should create a read-only custom control and an editable control for raw binary data. Possibly showing a preview of bytes in hex. but this will need to be prioritized and worked on later.